backends/bevy_picking_sprite/src/lib.rs

```diff
@@ -98,11 +98,13 @@ pub fn sprite_picking(
                         .or_else(|| images.get(image).map(|f| f.size().as_vec2()))?;
                     let anchor = sprite.anchor.as_vec();
                     (extents, anchor)
-                } else if let Some((sprite, atlas)) = sprite.1.zip(image.1) {
+                } else if let Some((sprite, atlas)) = sprite.0.zip(image.1) {
```

Comment:
This change should be reviewed

`sprite.0` appears to return `Sprite` 
`sprite.1` would redundantly return `TextureAtlas`